jdi-light-examples/src/main/java/selenium/seleniumPO/HomePage.java

```diff
@@ -39,6 +41,5 @@ public void navigateTo(String text) {
     }
 
     // TODO fix performance
-    @UI("//*[@ui='label']//*[contains(text(),'%s')]") public static WebList leftNavigation;
     //@UI("[ui=label][*'%s']") public static WebList leftNavigation;
```

Comment:
Guess we should move this comment too.
```
@FindBy(css = ".profile-photo [ui=label]")protected WebElement userName;
@FindBy(css = ".sidebar-menu [ui=label]")private List<WebElement> navigation;

// TODO fix performance
@UI("//*[@ui='label']//*[contains(text(),'%s')]") public static WebList leftNavigation;
//@UI("[ui=label][*'%s']") public static WebList leftNavigation;
```
I suggest to rearange lines something like that

Reply:
done:

public class HomePage {

    private String url = "https://jdi-testing.github.io/jdi-light/index.html";
    @FindBy(css = ".profile-photo")protected WebElement profilePhoto;
    @FindBy(css = ".profile-photo [ui=label]")protected WebElement userName;
    //@UI("[ui=label][*'%s']") public static WebList leftNavigation;
    @UI("//*[@ui='label']//*[contains(text(),'%s')]") public static WebList leftNavigation;
    @FindBy(css = ".sidebar-menu [ui=label]")private List<WebElement> navigation;